src/domain/prices/entities/asset-price.entity.ts

```diff
@@ -0,0 +1,3 @@
+export interface AssetPrice {
```

Comment:
What is the reason to not have an AJV schema for the `AssetPrice`?

Reply:
Yeah, we can include one, thanks.

Reply:
Done [17040f3](https://github.com/safe-global/safe-client-gateway/pull/732/commits/17040f3f07ad63a3682ce6b665dc451dfae545a2)